account/manager.go

```diff
@@ -165,13 +166,44 @@ func actionTxLabel(account *Account, action Action, isExpirySpend bool,
 	if err != nil {
 		log.Errorf("Internal error: failed to serialize json "+
 			"from %v", label)
-		return fmt.Sprintf(prefixTag, action)
+		return fmt.Sprintf(txLabelPrefixTag, action)
 	}
 
-	labelString := fmt.Sprintf(prefixTag, labelJson)
+	labelString := fmt.Sprintf(txLabelPrefixTag, labelJson)
 	return labelString
 }
 
+// IsPoolTx returns true if the given transaction is related to
+// pool.
+func IsPoolTx(tx *lnrpc.Transaction) bool {
+	return strings.HasPrefix(tx.Label, txLabelPrefixTag)
+}
+
+// ParseTxLabel parses and returns data fields stored in a given account
+// transaction label.
+func ParseTxLabel(label string) (*AccountTxLabel, error) {
+	label = strings.TrimPrefix(label, txLabelPrefixTag)
```

Comment:
I don't think this will work since the tag also contains the `%s`. So probably just need a `fmt.Sprintf(txLabelPrefixTag, "")` instead of the constant directly.

Reply:
Updated commit includes your suggested change and a unit test.